src/components/ConfirmationPage.tsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
-import type {TextStyle} from 'react-native';
+import type {TextStyle, ViewStyle} from 'react-native';
 import {View} from 'react-native';
 import useThemeStyles from '@hooks/useThemeStyles';
+import isIllustrationLottieAnimation from '@libs/isIllustrationLottieAnimation';
+import type IconAsset from '@src/types/utils/IconAsset';
 import Button from './Button';
 import FixedFooter from './FixedFooter';
+import ImageSVG from './ImageSVG';
 import Lottie from './Lottie';
 import LottieAnimations from './LottieAnimations';
 import type DotLottieAnimation from './LottieAnimations/types';
 import Text from './Text';
 
 type ConfirmationPageProps = {
     /** The asset to render */
-    animation?: DotLottieAnimation;
+    animation?: DotLottieAnimation | IconAsset;
```

Comment:
This name seems a little confusing when the 'animation' is not an animation. Can we change it to something that would match 'icon' and 'animation' for example something like 'asset'.

Reply:
We use `illustration` name in the `Section` component, so I think we should follow this pattern. Going to change it.